homeassistant/components/profiler/__init__.py

```diff
@@ -100,6 +96,8 @@ def _safe_repr(obj: Any) -> str:
             return f"Failed to serialize {type(obj)}"
 
     def _dump_log_objects(call: ServiceCall) -> None:
+        import objgraph  # pylint: disable=import-outside-toplevel
```

Comment:
let's add a comment when we inline imports

Reply:
87da23b74f7b3cd9c47b3b6a0574f268bfca9e98